.github/ISSUE_TEMPLATE/minor-release.md

```diff
@@ -35,21 +35,21 @@ Change the `master` branch to the next MINOR+1 version:
 - [ ] Bump the `bump-dev-MAJOR-MINOR+1` branch to the next development MINOR+1 version.
```

Comment:
is it intentional we dont have `<CRATE>` in the branch name here?
maybe we can also change `bump-dev` to `bump-to-MAJOR..`

Reply:
this was what was in the old checklist, and I figured it was OK since it's a branch that's not ending up in the upstream repo but just one's personal fork to be merged in a PR, but I can adopt.

I don't know that adding `to` is semantically relevant. Would there ever be a `bump-from...`?

Reply:
what do u mean when u say `bump-dev-..`? as in take dev to prod?
we can just use `bump-..`

Reply:
adopted